cclib/attribute_parsers/ccenergies.py

```diff
@@ -34,9 +35,33 @@ def psi4(file_handler, ccdata) -> Optional[dict]:
             return {ccenergies.__name__: np.array(this_ccenergies)}
         return None
 
-        # The geometry convergence targets and values are printed in a table, with the legends
+    @staticmethod
+    def gaussian(file_handler, ccdata) -> Optional[dict]:
+        line = file_handler.last_line
+        if getattr(ccdata, "ccenergies") is None:
```

Comment:
logic between 41-44 seems weird. do we need to check if it already exists if we're just rewriting it?

Reply:
Yeah this is a little weird but I think the logic is that in the future 44 could possibly append to existing `ccenergy`

Reply:
done!